common/version/version.go

```diff
@@ -5,7 +5,7 @@ import (
 	"runtime/debug"
 )
 
-var tag = "v4.4.52"
+var tag = "v4.4.54"
```

Comment:
```suggestion
var tag = "v4.4.53"
```

Reply:
last merge didn't bump version but taged `v4.4.53`here:
https://github.com/scroll-tech/scroll/tree/v4.4.53.

so I think we better skip to `v4.4.54`.